src/tla/Apalache.tla

```diff
@@ -38,6 +38,18 @@ x := e == x = e
  *)
 Gen(size) == {}
 
+(**
+ * A non-deterministic version of CHOOSE x \in S: TRUE.  That is, whenever
```

Comment:
I think direct definitions, not in terms of another operator, are easier to understand.

```suggestion
   * Non-deterministically pick a value out of the set `S`, if `S` is non-empty. If `S` is empty, return some value of the proper type.
 * This can be understood as a non-deterministic version of CHOOSE x \in S: TRUE. That is, whenever
```